Tests/Test Helpers/TestImage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using UnityEngine.UI;
+
+// if you create a custom class in a file that doesn't match the class name,
+// after the tests run Unity will complain that the compiler that imported 
+// "TestImage" isn't available anymore, so they go in their own files
+namespace UnityEditor.Experimental.EditorVR.Tests.TestHelpers
+{
+    public class TestImage : Graphic
```

Comment:
which rules ?  basically, i just needed a `Graphic` of some sort to test with inside `GetMaterialClone_ClonesGraphicMaterial` , but you have to drive from `Graphic`, can't instantiate one directly, and wanted a class i knew wouldn't change outside of the test code.